CHANGELOG.md

```diff
@@ -70,6 +70,42 @@ $govuk-suppressed-warnings: (
 
 This was added in [#2911 Add warning suppression functionality](https://github.com/alphagov/govuk-frontend/pull/2911)
 
+#### Configure components in JavaScript
+
+JavaScript components receiving configuration through data attributes now accept the same configuration options when instantiated in JavaScript. These components are:
+
+- the `Button` component, for its `preventDoubleClick` option (matching `data-prevent-double-click`)
+- the `CharacterCount` component, for its `maxlength`,`maxwords` and `threshold` options (matching `data-maxlength`, `data-maxwords` and `data-threshold`, respectively)
+- the `ErrorSummary` component, for its `disableAutoFocus` option (matching `data-disable-auto-focus`)
+- the `NotificationBanner` component, for its `disableAutoFocus` option (matching `data-disable-auto-focus`)
+
+You can omit these options when calling the Nunjucks macro and provide them:
+
+- at component instantiation, in a configuration object as second argument, for example:
+
+  ```js
```

Comment:
It would be better if the examples were not part of the bullet list.

Reply:
Oh, I thought it would help to keep them tied to the situation they give an example for. Is it because they break the flow when reading through the list? 

I've pushed an [updated version addressing this and the previous comment](https://github.com/alphagov/govuk-frontend/blob/8cd406c51964f988557bbee47513dc432e42d170/CHANGELOG.md#configure-components-in-javascript). (Here's the [old version for comparison](https://github.com/alphagov/govuk-frontend/blob/ac8e7820f3150452bf2ff2c50a5c20877db1cefb/CHANGELOG.md#configure-components-in-javascript)).